cai_causal_graph/time_series_causal_graph.py

```diff
@@ -152,17 +152,6 @@ def __ne__(self, other: object) -> bool:
         """Check if the graph is not equal to another graph."""
         return not (self == other)
 
-    def copy(self, include_meta: bool = True) -> TimeSeriesCausalGraph:
```

Comment:
Just confirming there is a unit test for `TimeSeriesCausalGraph` copying just to ensure this doesn't break it?

Reply:
Sorry saw that there is one once I made it to the end of the PR.